taskcluster/ci/browsertime/kind.yml

```diff
@@ -37,7 +37,7 @@ job-defaults:
                     subject: '[{product_name}] Raptor-Browsertime job "{task_name}" failed'
                     to-addresses: [perftest-alerts@mozilla.com]
             default: {}
-    run-on-tasks-for: []
+    run-on-tasks-for: [github-pull-request]
```

Comment:
You'll need to remove this before landing, otherwise the patch is good now

Reply:
removed